test/integration/taxonomy-integration.ts

```diff
@@ -0,0 +1,404 @@
+import { getTestOrganizationId, initPlainClient } from '../helpers'
+import { afterEach, beforeEach, describe, test } from 'mocha'
+import { expect } from 'chai'
+import { ConceptProps, CreateConceptProps } from '../../lib/entities/concept'
+import { ConceptSchemeProps, CreateConceptSchemeProps } from '../../lib/export-types'
+
+let conceptsToDelete: ConceptProps[] = []
+let conceptSchemesToDelete: ConceptSchemeProps[] = []
+
+describe('Taxonomy Integration', () => {
+  const client = initPlainClient({
+    organizationId: getTestOrganizationId(),
+  })
+
+  beforeEach(() => {
+    conceptsToDelete = []
+    conceptSchemesToDelete = []
+  })
+
+  afterEach(async () => {
+    for (const concept of conceptsToDelete) {
+      const conceptToBeDeleted = await client.concept.get({
+        conceptId: concept.sys.id,
+      })
+      await client.concept.delete({
+        conceptId: conceptToBeDeleted.sys.id,
+        version: conceptToBeDeleted.sys.version,
+      })
+    }
+
+    for (const conceptScheme of conceptSchemesToDelete) {
+      const conceptSchemeToBeDeleted = await client.conceptScheme.get({
+        conceptSchemeId: conceptScheme.sys.id,
+      })
+      await client.conceptScheme.delete({
+        conceptSchemeId: conceptSchemeToBeDeleted.sys.id,
+        version: conceptSchemeToBeDeleted.sys.version,
+      })
+    }
+  })
+
+  test('delete a concept', async () => {
+    const concept: CreateConceptProps = {
+      prefLabel: {
+        'en-US': 'Test Concept',
+      },
+    }
+    const result = await client.concept.create({}, concept)
+
+    isConceptProps(result)
+    expect(result.prefLabel['en-US']).to.equal('Test Concept')
+    expect(result.uri).to.null
+
+    await client.concept.delete({
+      conceptId: result.sys.id,
+      version: result.sys.version,
+    })
```

Comment:
Is there nothing we want to assert after performing the deletion? probably at minimum a successful status code?

Reply:
I added a test to be sure the item is deleted.

Reply:
Nice. Do you think we should check the status code for the DELETE too?